README.md

```diff
@@ -1 +1,105 @@
-# portfolio
\ No newline at end of file
+<a name="readme-top"></a>
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="https://raw.githubusercontent.com/microverseinc/readme-template/master/murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
```

Comment:
- [ ] This is the Microverse README logo. kindly consider removing this logo. you can add your own logo if you have one. it will make your readme more professional.